src/lib.rs

```diff
@@ -238,15 +238,22 @@ impl<B: MysqlShim<W>, R: Read, W: Write> MysqlIntermediary<B, R, W> {
     fn init(&mut self) -> Result<(), B::Error> {
         self.writer.write_all(&[10])?; // protocol 10
 
+        let mut capabilities = CapabilityFlags::empty();
+        capabilities.insert(CapabilityFlags::CLIENT_PROTOCOL_41);
+        capabilities.insert(CapabilityFlags::CLIENT_RESERVED);
+        capabilities.insert(CapabilityFlags::CLIENT_SECURE_CONNECTION);
```

Comment:
Do we actually support secure connections? I did not think so?

Reply:
We don't. But those two are the minimal sets of flags that fixed the [connection phase](https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_connection_phase.html) issue with the latest Connector/J. But I have to fix the client authentication problem.